rush.json

```diff
@@ -1,483 +0,0 @@
-   * for details about these alternatives.
-   */
-  "pnpmVersion": "2.19.2",
-  // "npmVersion": "5.4.0",
-  // "yarnVersion": "1.9.4",
-  /**
-   * Options that are only used when the PNPM package manager is selected
-   */
-  "pnpmOptions": {
-    /**
-     * If true, then Rush will add the "--strict-peer-dependencies" option when invoking PNPM.
-     * This causes "rush install" to fail if there are unsatisfied peer dependencies, which is
-     * an invalid state that can cause build failures or incompatible dependency versions.
-     * (For historical reasons, JavaScript package managers generally do not treat this invalid
-     * state as an error.)
-     *
-     * The default value is false to avoid legacy compatibility issues.
-     * It is strongly recommended to set strictPeerDependencies=true.
-     */
-    "strictPeerDependencies": true
-  },
-  "repository": {
-    /**
-     * This setting is sometimes needed when using "rush change" with multiple Git remotes.
-     * It specifies the remote url for the official Git repository.  If this URL is provided,
-     * "rush change" will use it to find the right remote to compare against.
-     */
-    "url": "https://github.com/OfficeDev/office-ui-fabric-react.git"
-  },
-  /**
-   * Event hooks are customized script actions that Rush executes when specific events occur
-   */
-  "eventHooks": {
-    /**
-     * The list of shell commands to run before the Rush installation starts
-     */
-    "preRushInstall": [
-      // "common/scripts/pre-rush-install.js"
-    ],
-    /**
-     * The list of shell commands to run after the Rush installation finishes
-     */
-    "postRushInstall": [],
-    /**
-     * The list of shell commands to run before the Rush build command starts
-     */
-    "preRushBuild": [],
-    /**
-     * The list of shell commands to run after the Rush build command finishes
-     */
-    "postRushBuild": []
-  },
-  /**
-   * Older releases of the NodeJS engine may be missing features required by your system.
-   * Other releases may have bugs.  In particular, the "latest" version will not be a
-   * Long Term Support (LTS) version and is likely to have regressions.
-   *
-   * Specify a SemVer range to ensure developers use a NodeJS version that is appropriate
-   * for your repo.
-   */
-  "nodeSupportedVersionRange": ">=8.9.0",
-  /**
-   * If you would like the version specifiers for your dependencies to be consistent, then
-   * uncomment this line. This is effectively similar to running "rush check" before any
-   * of the following commands:
-   *
-   *   rush install, rush update, rush link, rush version, rush publish
-   *
-   * In some cases you may want this turned on, but need to allow certain packages to use a different
-   * version. In those cases, you will need to add an entry to the "allowedAlternateVersions"
-   * section of the common-versions.json.
-   */
-  "ensureConsistentVersions": true,
```

Comment:
With lerna/yarn do we have any mechanism for enforcing consistent dependency versions? Would be kind of a shame to lose that.